herepy/here_enum.py

```diff
@@ -6,15 +6,19 @@ class RouteMode(Enum):
     """Modes which is used in routing api functions."""
 
     fastest = 'fastest'
+    shortest = 'shortest'
+    balanced = 'balanced'
     bicycle = 'bicycle'
     car = 'car'
+    car_hov = 'carHOV'
+    traffic_enabled = 'traffic:enabled'
     traffic_disabled = 'traffic:disabled'
+    traffic_default = 'traffic:default'
```

Comment:
Thanks for adding new types 👍 could you please also add some basic tests on `test_enum.py`.

Reply:
Done